### ISSUE TITLE
ARM Support ( with patch )

### ISSUE THREAD

**Issue by brontide** [tarampampam/3proxy-docker#30]
The following minimal modifications allow the repo to build on aarch64 or x86_64.  I am running this on Oracle Cloud arm instances. I have also tested the changes on x86_64 and it continues to build without error.  Given mustpl support the arch ppc64le and s390x may also work but that is not tested.

```
diff --git a/Dockerfile b/Dockerfile
index fc22573..59957da 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -44,10 +44,10 @@ RUN set -x \
     && mkdir -p ./etc ./bin ./usr/local/3proxy/libexec ./etc/3proxy \
     && echo '3proxy:x:10001:10001::/nonexistent:/sbin/nologin' > ./etc/passwd \
     && echo '3proxy:x:10001:' > ./etc/group \
-    && wget -O ./bin/dumb-init "https://github.com/Yelp/dumb-init/releases/download/v1.2.5/dumb-init_1.2.5_x86_64" \
+    && wget -O ./bin/dumb-init "https://github.com/Yelp/dumb-init/releases/download/v1.2.5/dumb-init_1.2.5_$(arch)" \
     && chmod +x ./bin/dumb-init
 
-COPY --from=builder /lib/x86_64-linux-gnu/libdl.so.* ./lib/
+COPY --from=builder /lib/*-linux-gnu/libdl.so.* ./lib/
 COPY --from=builder /tmp/3proxy/bin/3proxy ./bin/3proxy
 COPY --from=builder /tmp/3proxy/bin/*.ld.so ./usr/local/3proxy/libexec/
 COPY --from=ghcr.io/tarampampam/mustpl:0.1.0 /bin/mustpl ./bin/mustpl
```

**Comment by tarampampam** [comment]
@brontide Thanks for your suggestion, very cool! The release will be published shortly 😄

**Comment by tarampampam** [comment]
You can try it now - `ghcr.io/tarampampam/3proxy:1.8.1`

**Comment by tarampampam** [comment]
please, do **NOT** use 1.8.1 (the reason [described here](https://github.com/tarampampam/3proxy-docker/issues/32)). I have released `1.8.2` there all works fine

**Comment by brontide** [comment]
Tested 1.8.2 on my instance and it just drops in and works.
